Gopkg.toml

```diff
@@ -52,7 +52,7 @@
 
 [[constraint]]
   name = "github.com/intel/govmm"
-  revision = "e82e8498c5a214b24ac75e0a05ace556bf91a9ab"
+  revision = "110d2fa0493db5992c3e0ab89d256576bf5ee86d"
```

Comment:
Do you also need to update the govmm/qemu/qemu.go file in the vendor directory too?